### PR TITLE
Added fields volume_profiling_flag and volume_profiling_state

### PR DESCRIPTION
**Pull request by shtripat**
Added the fields volume_profiling_flag and volume_profiling_state
for cluster object

tendrl-bug-id: Tendrl/gluster-integration#553
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by shtripat** [comment]
@a2batic @nthomas-redhat @r0h4n @GowthamShanmugam @gnehapk plz review

**Comment by GowthamShanmugam** [comment]
LGTM

**Comment by r0h4n** [comment]
Could you please fix the vulture report too https://travis-ci.org/Tendrl/commons/jobs/343844783

@shtripat

**Comment by r0h4n** [comment]
@shtripat In the vulture report https://travis-ci.org/Tendrl/commons/jobs/344273070

It clearly says "tendrl/commons/__init__.py:562: unused variable 'importer' "  , which means you need to either use the variable or remove it since it is not used.

Same for "unused function" and "unused attribute" , just remove the function or attribute if it is not really used much.

**Comment by shtripat** [comment]
@r0h4n so these kind issues could be a lot in projects and fixing all of them as part this PR, not sure how feasible or good that is. For new sources I can take care at the moment on this PR but for all I feel we should do a separate task only for these kind of cleanups. What you say?

**Comment by r0h4n** [comment]
@shtripat  Ack https://github.com/Tendrl/commons/issues/808

File: tox.ini [hunk 1]
```diff
@@ -41,6 +41,11 @@ deps =
     vulture
 commands = vulture tendrl
 
+[testenv:prospector]
```
Comment (r0h4n) [inline tox.ini]: please make a new PR for this along with an issue. Cant take this in this PR

Comment (shtripat) [reply tox.ini]: Done

**Pull request merged**